src/main/java/step4/Race.java

```diff
@@ -0,0 +1,43 @@
+package step4;
+
+import java.util.Random;
+
+public class Race {
+    private int numOfCar;
+    private int numOfCycle;
+    private InputView inputView = new InputView();
+    private ResultView resultView = new ResultView();
+    private CarManager[] carManagers;
```

Comment:
전체적으로 `array` 를 사용하여서 `array` 컨트롤을 하기 위해 반드시 `for (int i = 0; ` 등이 반복 되고 있습니다 😄 
물론 `array` 도 나쁘지 않지만 아무래도 `for (int i = 0` 로 제어를 하다보니 반복적으로 `for` 문이 등장하고 
`index`를 잘못 넣거나 하는 사람 실수(?) 등을 유발 할 수도 있습니다 😄 

```suggestion
    private List<CarManager> carManagers
```
와 `for-each`를 이용하여 한번 개선해보시면 좋을 것 같습니다 😄 
이 부분도 검토 부탁 드립니다 🙇